distributed-tests/rackspace-server-manager.py

```diff
@@ -55,24 +58,19 @@ def create_node(nova, counts):
             print 'No IP address assigned!'
             sys.exit(1)
         else:
-            ips.append(ip_address)
+            ips.update({str(ip_address):False})
 
-
-    for ip in reversed(ips):
-        ret = ping_node(ip)
-        timeout = time.time()
-        while ret != 0:
-            if time.time() > timeout:
-                print 'Not able to connect to a server with IP address {0}'.format(ip)
-                ips = ips.remove(ip)
-                break
-            time.sleep(5)
-            ret = ping_node(ip)
-
-    print 'The list of alive servers is: {0}'.format(ips)
+    timeout = 0
+    ret = check_ssh(ips)
+    while ret != True and timeout < 300:
+        ret = check_ssh(ips)
+        time.sleep(5)
+        timeout = timeout + 5
+    reachable_ip = [ip for ip, val in ips.items() if val==True]
+    print 'The list of reachable servers: {0}'.format(ip)
```

Comment:
Might want to print the unreachable Ips instead.

Reply:
May need to print reachable Ips also in the console logs so as to further debug easily.